src/smufl_arpeggio_fix.lua

```diff
@@ -0,0 +1,35 @@
+function plugindef()
+  finaleplugin.RequireSelection = false
```

Comment:
FYI, if `finaleplugin.RequireSelection` is `false`, you don't need this line of code.